pkg/controllers/runtime_controller.go

```diff
@@ -64,12 +67,20 @@ func NewRuntimeReconciler(reconciler RuntimeReconcilerInterface, client client.C
 
 // ReconcileInternal handles the logic of reconcile runtime
 func (r *RuntimeReconciler) ReconcileInternal(ctx cruntime.ReconcileRequestContext) (ctrl.Result, error) {
-	// 1.Get the runtime
+	// 0.Get the runtime
```

Comment:
I think starting with `1` is a better choice because it conforms to existing code style.